src/commands/setupbash.sh

```diff
@@ -64,11 +64,11 @@ installDepend() {
     if [[ $PACKAGER == "pacman" ]]; then
         if ! command_exists yay && ! command_exists paru; then
             echo "Installing yay as AUR helper..."
-            sudo ${PACKAGER} --noconfirm -S base-devel
+            ${PACKAGER} --noconfirm -S base-devel
```

Comment:
```suggestion
            sudo ${PACKAGER} --noconfirm -S base-devel
```

We still need to use sudo